pom.xml

```diff
@@ -22,6 +23,11 @@
       <artifactId>spring-boot-starter</artifactId>
       <version>${spring-boot.version}</version>
```

Comment:
remove everywhere, spring boot starter parent provides versions

```suggestion
```

Reply:
Oh, so the question why you don't have it? ) 

```
<parent>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-parent</artifactId>
```

Reply:
So, your response is that you don’t see it in the POM, yet you suggest removing the versions?) OK, I've added parent.